test/integration/package_int_test.go

```diff
@@ -783,6 +784,56 @@ func (suite *PackageIntegrationTestSuite) TestChangeSummary() {
 	cp.ExpectExitCode(0)
 }
 
+func (suite *PackageIntegrationTestSuite) TestBuildInProgress() {
+	suite.OnlyRunForTags(tagsuite.Package)
```

Comment:
Let's move this to `runtime_int_test.go`. I don't think package is appropriate, this isn't doing anything with `state packages`.